config/sphinx_local.yaml

```diff
@@ -9,6 +9,8 @@ theme:
   google_analytics: 'UA-7301842-8'
   nav_excluded:
     - /
+    - /troubleshooting
```

Comment:
Do you think we need this landing page? I've seen them throughout docs before, but not sure if they are standard or anything like that.

Reply:
We don't really have a standardized practice here. I was thinking it may be useful if we eventually add other types of Troubleshooting pages (in addition to Connection Troubleshooting). However I agree that right now it looks a bit weird with just one page. I can update to make it a single top-level page. If we move this to a single page, it may make sense to move this to a subpage under Connect to MongoDB. What are your thoughts?

Reply:
Let me make sure I understand:

It's required that all sections in the left hand side of our docs have an associated landing page. It's not possible to just click "Troubleshooting" and have that expand to show the subpages. Is that right? 

This makes sense to me:

> I was thinking the top-level landing page may be useful if we eventually add other types of Troubleshooting pages (in addition to Connection Troubleshooting). However I agree that right now it looks a bit weird with just one page. I can update to make it a single top-level page.

This part doesn't make sense to me:

> If we move this to a single page, it may make sense to move this to a subpage under Connect to MongoDB.

I think we'll need an explicit section for troubleshooting. Shall we discuss IRL?

Reply:
Discussed IRL. Leaving as-is for now since we need a landing page for top-level pages. Going to remove the previous / next page arrows. This will look better once we add a second page.